04-single-hop-6lowpan-icmp/04-single-hop-6lowpan-icmp.md

```diff
@@ -132,3 +132,22 @@ a samr21-xpro node.
 ### Result
 
 <10% packets lost on the pinging node.
+
+Task #09 - ICMPv6 stress test
+=============================
+### Description
+
+Rapid ICMPv6 echo request/reply exchange from two iotlab-m3 nodes simultaneously
+to one iotlab-m3.
+* Stack configuration:    6LoWPAN (default)
+* Channel:                26
+* Count:                  200
+* Interval:               0ms
+* Payload:                1232B
+* Destination Address:    Link local unicast (fe80::.../64)
+
+### Result
+
+All nodes are still running, reachable, and the packet buffer is empty 3 seconds
```

Comment:
3 seconds because that is the time a fragmented datagram might stay in 6LoWPAN's reassembly buffer before completion is given up.

Reply:
How do you determine that the packet buffer is empty ? I tried `pktbuf` and it returns:
```
> pktbuf
pktbuf
packet buffer: first byte: 0x20001b18, last byte: 0x20003318 (size: 6144)
  position of last byte used: 1672
~ unused: 0x20001b18 (next: 0, size: 6144) ~
```
I guess the `unused` means it's empty ?